blues/application/tasks.py

```diff
@@ -224,24 +224,54 @@ def generate_nginx_conf(role='www'):
         f.write(conf)
 
 
-def notify_deploy(role=None):
+def get_github_org():
+    from .project import git_repository
+
+    url = git_repository()
+    if 'url' not in url or not url.startswith('git@github.com:'):
+        return None
+
+    domain, repo = url.split(':')
+    org, repo = repo.split('/')
+    return org
+
+
+def notify_deploy(role=None, commits=None):
     from .project import project_name, git_repository_path
 
+    msg = '`{deployer}` deployed '
+    if role:
+        msg += '`{project}::{state}:{role}` '
```

Comment:
Not sure if we need double colon `::`. Can just single colon work here?

You can simplify a bit with:

```py
'role': role or '',
```

so we can skip `if`. I have a feeling that you made a typo here though, but I actually prefer this typo :smiley_cat: The only change I would prefer is swapping places of `role` and `state`.